src/routes/email/email.controller.delete-email.spec.ts

```diff
@@ -74,48 +74,48 @@ describe('Email controller delete email tests', () => {
     const chain = chainBuilder().build();
     const timestamp = jest.now();
     const privateKey = generatePrivateKey();
-    const account = privateKeyToAccount(privateKey);
-    const accountAddress = account.address;
+    const signer = privateKeyToAccount(privateKey);
+    const signerAddress = signer.address;
     const safeAddress = faker.finance.ethereumAddress();
-    const email = emailBuilder()
-      .with('account', accountAddress)
+    const account = accountBuilder()
+      .with('signer', signerAddress)
       .with('safeAddress', safeAddress)
       .with('chainId', chain.chainId)
       .build();
-    const message = `email-delete-${chain.chainId}-${safeAddress}-${accountAddress}-${timestamp}`;
-    const signature = await account.signMessage({ message });
-    emailDatasource.getEmail.mockResolvedValue(email);
-    emailDatasource.deleteEmail.mockImplementation(() => Promise.resolve());
+    const message = `email-delete-${chain.chainId}-${safeAddress}-${signerAddress}-${timestamp}`;
+    const signature = await signer.signMessage({ message });
+    accountDataSource.getAccount.mockResolvedValue(account);
+    accountDataSource.deleteAccount.mockImplementation(() => Promise.resolve());
     emailApi.deleteEmailAddress.mockResolvedValue();
 
     await request(app.getHttpServer())
       .delete(`/v1/chains/${chain.chainId}/safes/${safeAddress}/emails`)
       .send({
-        account: account.address,
+        account: signer.address,
```

Comment:
We should also change this to `signer` if we are doing so elsewhere.

Reply:
[d56dd44](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/d56dd4443b128abb5be9c79161c5d9b8bcc4b8ed)